Analyse.cs

```diff
@@ -53,4 +59,21 @@ public Analyse(Input input)
             }
         }
     }
+
+    /// <summary>
+    /// Finds all words longer than 7 letters.
+    /// </summary>
+    /// <param name="input">The Input object to review.</param>
+    /// <returns>An array of words longer than 7 letters.</returns>
+    private List<string> GetLongWords(Input input)
+    {
+        List<string> longWords = 
+            (
+            from word in Regex.Matches(input.InputText, "[a-z']{8,}")
+            where word.Value.Length > 7
+            select word.Value
+            ).ToList(); // Converts from IEnumerable<string> to List<string>.
```

Comment:
Redundant `where` statement on line 73.
```suggestion
        List<string> longWords = 
            (
            from word in Regex.Matches(input.InputText, "[a-z']{8,}")
            select word.Value
            ).ToList(); // Converts from IEnumerable<string> to List<string>.
```